CMSgov/mint-app
# [EASI-3437] - Search pagination fix

# EASI-3437

## Changes and Description

- Changed `react-table` `useTable` config `autoResetPage` to `true`

This value should be set to false only if data updates from an external source and pagination shouldn't be reset
This is neither the case where we implement global filtering in MINT

## How to test this change

- Seed and add enough model plans to render a page 2 on the `/models` tab.  Verify that while on page 2, and a global filter text is entered, that the page resets to 1

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
